src/library.js

```diff
@@ -1488,8 +1488,15 @@ LibraryManager.library = {
 #endif
 
   __cxa_pure_virtual: function() {
+#if !MINIMAL_RUNTIME
     ABORT = true;
+#endif
+
+#if MINIMAL_RUNTIME && !ASSERTIONS
+    throw 'pv';
```

Comment:
I think a better microoptimization would be to replace this throw with `abort()`. That will be minified into `x()`, just 3 characters, versus 10.

(If abort() doesn't exist in the MINIMAL_RUNTIME path, it can be added; the JS optimizer will remove it if it isn't used.)

Reply:
Yeah, good point - switched to `abort()` instead.

Reply:
In MINIMAL_RUNTIME, `abort()` is a JS library function, so can be aliased to in here.